src/pages/RecipesList.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import ThemeProvider from 'react-bootstrap/ThemeProvider';
+import { v4 as uuidv4 } from 'uuid';
+import RecipeRow from '../components/RecipeRow';
+
+function RecipesList() {
```

Comment:
Kindly make sure that you have a `back` button on the details page that take users back to the homepage as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/188237324-c5076d9d-3d57-45c7-b0e3-68ac50c54fd6.png)

- So far the application relies on the browser's back button for navigation. The should be an in-app back button as well.